README.md

```diff
@@ -135,7 +135,7 @@ ABCDs Detector MVP supports a single video evaluation for the following features
   - Call To Action (Text)
   - Call To Action (Speech)
 
-For a definition for each of these signals please take a look at the [features.py](https://github.com/google-marketing-solutions/abcds-detector/blob/main/feature_configs/features.py) file.
+For a definition for each of these signals please take a look at the [long_form_abc_features.py](https://github.com/donaldseaton/abcds-detector/blob/main/features_repository/long_form_abcd_features.py) and [shorts_features.py](https://github.com/donaldseaton/abcds-detector/blob/main/features_repository/shorts_features.py) files.
```

Comment:
Hi Donald, thanks so much for contributing! This link is referencing your fork donaldseaton, could you please change that? Thanks!